src/conn/conn_sweep.c

```diff
@@ -8,6 +8,10 @@
 
 #include "wt_internal.h"
 
+#define	WT_DHANDLE_CAN_DISCARD(dhandle)					\
```

Comment:
Maybe move to dhandle.h - possibly can be combined in part with WT_DHANDLE_INACTIVE.